keras/backend/mxnet_backend.py

```diff
@@ -2902,7 +2957,29 @@ def sparse_categorical_crossentropy(target, output, from_logits=False, axis=-1):
     # Returns
         Output tensor.
     """
-    raise NotImplementedError('MXNet Backend: Sparse operations are not supported yet.')
+    output_dimensions = list(range(ndim(output)))
+    if axis != -1 and axis not in output_dimensions:
+        raise ValueError(
+            '{}{}{}'.format(
+                'Unexpected channels axis {}. '.format(axis),
+                'Expected to be -1 or one of the axes of `output`, ',
+                'which has {} dimensions.'.format(len(int_shape(output)))))
+
+    mx_output = output.symbol
+    # scale predictions so that the class probas of each sample sum to 1
```

Comment:
see comment above about writing the docstring - https://www.python.org/dev/peps/pep-0257/#one-line-docstrings

Reply:
@kalyc compare to element-wise comparison operators (e.g. K.equal), sparse_categorical_crossentropy  and categorical_crossentropy are more complicated, and it requires different backend to do different processing logic. It's important to place the inline comments.  Placing these comments in doc string will confuse users. See tensorflow_backend.py for reference.

Reply:
Thanks for the explanation!
[minor] change probas to probabilities